tests/dla/test_adjoint_repr.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for pennylane/dla/adjoint_repr.py functionality"""
+import pytest
+import numpy as np
+
+import pennylane as qml
+
+from pennylane.dla import adjoint_repr
+from pennylane.pauli import PauliWord, PauliSentence
+
+n = 3
+gens = [PauliSentence({PauliWord({i: "X", i + 1: "X"}): 1.0}) for i in range(n - 1)]
+gens += [PauliSentence({PauliWord({i: "Z"}): 1.0}) for i in range(n)]
+Ising3 = qml.dla.lie_closure(gens, pauli=True)
+
+
+class TestAdjointRepr:
```

Comment:
I think we should have a test where the generators are not single Pauli words, but true linear combinations. Maybe that could just be a second example, which is passed to the `test_adjoint_repr_elements` test?

Reply:
Good point! This unraveled a lot of things that still needed sorting out, in particular with regards to normalization. Added such a case now and all seems well 👍